src/chrome/content/rules/Andrew-Munsell.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Andrew Munsell">
+
+	<target host="andrewmunsell.com" />
+	<target host="www.andrewmunsell.com" />
+
+	<rule from="^http://(?:www\.)?andrewmunsell\.com/"
+		to="https://www.andrewmunsell.com/" />
```

Comment:
You should probably use the new ruleset format for these

<rule from="^http:"
to="https:" />